src/parser/transform/statement/transform_alter_table.cpp

```diff
@@ -19,13 +24,35 @@ vector<string> Transformer::TransformNameList(duckdb_libpgquery::PGList &list) {
 	return result;
 }
 
-unique_ptr<AlterStatement> Transformer::TransformAlter(duckdb_libpgquery::PGAlterTableStmt &stmt) {
+void AddToMultiStatement(const unique_ptr<MultiStatement> &multi_statement, unique_ptr<AlterInfo> alter_info) {
+	auto alter_statement = make_uniq<AlterStatement>();
+	alter_statement->info = std::move(alter_info);
+	multi_statement->statements.push_back(std::move(alter_statement));
+}
+
+void AddUpdateToMultiStatement(const unique_ptr<MultiStatement> &multi_statement, const string &column_name,
+                               const string &table_name, const unique_ptr<ParsedExpression> &original_expression) {
+	auto update_statement = make_uniq<UpdateStatement>();
+
+	auto table_ref = make_uniq<BaseTableRef>();
+
+	table_ref->table_name = table_name;
+	update_statement->table = std::move(table_ref);
+
+	auto set_info = make_uniq<UpdateSetInfo>();
+	set_info->columns.push_back(column_name);
+	set_info->expressions.push_back(original_expression->Copy());
+	update_statement->set_info = std::move(set_info);
+
+	multi_statement->statements.push_back(std::move(update_statement));
+}
+unique_ptr<SQLStatement> Transformer::TransformAlter(duckdb_libpgquery::PGAlterTableStmt &stmt) {
 	D_ASSERT(stmt.relation);
 	if (stmt.cmds->length != 1) {
 		throw ParserException("Only one ALTER command per statement is supported");
 	}
 
-	auto result = make_uniq<AlterStatement>();
+	auto result = make_uniq<MultiStatement>();
```

Comment:
I think this should stay an `AlterStatement`, as that's what makes the most sense for the base case.
Instead we can create the `MultiStatement` inside the block where we do the workaround, and return it there.